Branch-SDK/src/io/branch/referral/PrefHelper.java

```diff
@@ -185,6 +200,10 @@ static void shutDown() {
      * API uses.
      */
     public String getAPIBaseUrl() {
+        if (URLUtil.isHttpsUrl(customServerURL_)) {
+            return customServerURL_;
+        }
```

Comment:
I suspect the following lines will cause unit tests to fail if the `Build.VERSION.SDK_INT` is < 20

Reply:
It should still pass.  the < 20 is for TLS level supported by the url.  I can't know that for custom urls.

Reply:
Oh wait, you might be referring to running the test on an old device.  Yea that will fail.